docs/topics/circuit-breaker-mission-deploy-booster.adoc

```diff
@@ -72,6 +77,4 @@ NAME                     HOST/PORT
 +
 A pod's route information gives you the base URL which you use to access it. In the example above, you would use `http://{app-name}-{project-name}.{osl-route-hostname}` as the base URL to access the applications.
```

Comment:
I see that this is  not part of the diff, but we should avoid using the Saxon genitive in out docs. 
I'd change that to:

> The route information of a pod gives you the base URL that you use to access it. In the example above, you would use `http://{app-name}-{project-name}.{osl-route-hostname}` as the base URL to access the applications.